packages/manager/modules/account-sidebar/src/index.js

```diff
@@ -13,6 +14,13 @@ const moduleName = 'ovhManagerAccountSidebar';
 angular
   .module(moduleName, [ngAtInternet, paymentMean, shortcuts, userInfos])
   .component('ovhManagerAccountSidebar', component)
+  .run(
+    /* @ngInject */ ($rootScope, $transitions) => {
+      $transitions.onSuccess({ to: 'error' }, () => {
```

Comment:
Not all states are called `error` in the manager, and imo the accountSidebar component shouldn't be aware of the routing implementation. Maybe we should instead send an event when we transition to error page in the managers

Reply:
removed it and added the events in managers